src/app/test-record/vehicle/vehicle-edit/vehicle-edit.component.html

```diff
@@ -0,0 +1,184 @@
+<div class="govuk-grid-row">
```

Comment:
To check whether to display errors you should be checking for the valid or invalid status of the particular field NOT like this `[class.govuk-form-group--error]="formErrors?.includes('Enter the vin')`

Reply:
I have a function in test-record component that checks the **validity** of **each** field in the form and adds the error accordingly. Would you please check that? Thanks.